ereolen_support/ereolen_support.helper.inc

```diff
@@ -14,34 +14,39 @@
  * @return string|null
  *   Account id of the user.
  */
-function _ereolen_search_user(string $email): ?string {
-  $accountId = NULL;
-  $requestParts = _ereolen_support_get_request_parts('/user/search?query=' . $email, 'GET');
-  $searchQueryResult = _ereolen_support_callApi($requestParts);
-  // Endpoint searches in all fields, so we must ensure email match.
+function _ereolen_support_search_user(string $email): ?string {
+  $searchQueryResult = _ereolen_support_call_api('/user/search?query=' . $email, 'GET');
```

Comment:
You should always use https://www.php.net/manual/en/function.http-build-query.php (or something similar, e.g. https://api.drupal.org/api/drupal/includes%21common.inc/function/url/7.x) to build URLs with query strings, e.g.

```suggestion
  $searchQueryResult = _ereolen_support_call_api('/user/search' . http_build_query(['query' => $email], 'GET');
```